modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java

```diff
@@ -470,6 +502,7 @@ private String getAccessToken(String clientName, String username, String passwor
         org.wso2.carbon.automation.test.utils.http.client.HttpResponse tokenGenerateResponse = HttpRequestUtil
                 .doPost(tokenApiUrl, messageBody, accessKeyMap);
         JSONObject tokenGenJsonObject = new JSONObject(tokenGenerateResponse);
+        log.info("Token Response: " + tokenGenJsonObject.toString());
```

Comment:
Please call `String getResponseBody(HttpResponse response)` here and print only the token response body.